clusterloader2/testing/load/config.yaml

```diff
@@ -93,10 +94,12 @@ steps:
     Params:
       action: start
       replicasPerProbe: {{AddInt 2 (DivideInt .Nodes 100)}}
+  {{if $PROMETHEUS_SCRAPE_KUBE_PROXY}}
```

Comment:
Given that this measurement has no way to work if scraping kube-proxy is disabled, we should simply encode that in the measurement itself (it should be checking if we're scriaping kube-proxy and self-disable if we're not).

Reply:
I agree, created https://github.com/kubernetes/perf-tests/issues/807 for that.

Reply:
Let's do it in another commit.
(or find some solution to get that metric)